t04_refactored.py

```diff
@@ -213,9 +213,50 @@ def team_16_adv():
 ###################################################################################
 
 def team_17_adv():
-    pass
-    # TODO Add your code here
+    """
+    Creates two events that if the user replies with yes to either
+    to recieve a dog or sword, they survive and get a prize!
+    If they do not accept the dog, they die, but the sword is optional.
+    """
```

Comment:
There should be a Google doc link in your docstring:

In your function’s docstring, add a link to your team’s Google Doc (this document) as well as the names of all partners who worked on that function.